README.md

```diff
@@ -113,16 +113,16 @@ The structure of your behavior data should be as follows:
 
 ### Structure breakout
 
-In this example, the `comments`, `articles`, and `routes` keys represent what is referred to internally to the Behavior Service as "Behavior Groups".  Within these groups are the individual "Behaviors" that, in this structure, represent whether a user has permission to perform the listed behavior.  These same behavior key names can be used in other code, as discussed in the "Components" section, to further refine the desired behavior beyond just whether or not a user has permission.
+In this example, the `comments`, `articles`, and `routes` keys represent what is referred to internally to the Behavior Service as "Resources".  Within these groups are the individual "Activities" that, in this structure, represent whether a user has permission to perform the listed action.
 
-**NOTE:** *Except for the `route` key name it DOES NOT matter what you name the keys (either Behavior Groups or Behaviors) as they represent whatever CONCEPTS you want them to relate to in your application.*
+**NOTE:** *Except for the `route` key name it DOES NOT matter what you name the keys (Resources) as they represent whatever CONCEPTS you want them to relate to in your application.*
 
-**NOTE:** *Except for the `route` key name it DOES NOT matter what case you use for the keys (either Behavior Groups or Behaviors) though you will need to use the same case when referencing them in use of the Components or in Controllers.*
+**NOTE:** *Except for the `route` key name it DOES NOT matter what case you use for the keys (Resources) though you will need to use the same case when referencing them in use of the Components or in Controllers.*
```

Comment:
Maybe we should change "the Components or in Controllers" to something more generic and less "controller-promoting", such as "Services, Components, etc".

In either case the "the" before "Components" should be removed.